File: C++/C++Basic/04.Conditional-Statements-Exercise/02.bonusScore.cpp
#include <iostream>

using namespace std;

int main(){
    int startPoint;
    cin >> startPoint;

    double bonus = 0;
    
    if (startPoint <= 100){
        bonus = 5;
    }

    else if (startPoint > 1000){
        bonus = startPoint * 0.10;
    }

    else if (startPoint > 100){
        bonus = startPoint * 0.20;
    }

    if (startPoint % 2 == 0){
        bonus += 1;
    }
    else if (startPoint % 10 == 5){
        bonus += 2;
    }

    double totalBonus = bonus + startPoint;

    cout << bonus << endl;
    cout << totalBonus << endl;

    return 0;
}

File: Python/Basic/04.Conditional-Statements-Exercise/02.bonus_score.py
start_point = int(input())

bonus = 0
total_point = 0

if start_point <= 100:
    bonus = 5
elif start_point <= 1000:
    bonus = start_point * 0.20
elif start_point > 1000:
    bonus = start_point * 0.10

if start_point % 2 == 0:
    bonus += 1
elif start_point % 5 == 0:
    bonus += 2
total_point = start_point + bonus

print(bonus)
print(total_point)